pkgs/tools/system/hddfancontrol/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, python3Packages, fetchFromGitHub, hddtemp, hdparm, smartmontools }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "hddfancontrol";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "desbma";
+    repo = pname;
+    rev = version;
+    sha256 = "0b2grf98qnikayn18xll01dkm5pjpcjxdffgx1nyw9s0gqig8dg0";
+  };
+
+  propagatedBuildInputs = [
+    python3Packages.python-daemon
+    hddtemp
+    hdparm
+    smartmontools
+  ];
+
+  postInstall = ''
+    mkdir -p $out/etc/systemd/system
+    substitute systemd/hddfancontrol.service $out/etc/systemd/system/hddfancontrol.service \
+        --replace /usr/bin/hddfancontrol $out/bin/hddfancontrol
+    sed -i -e '/EnvironmentFile=.*/d' $out/etc/systemd/system/hddfancontrol.service
+  '';
+
+  meta = with lib; {
+    description = "Dynamically control fan speed according to hard drive temperature on Linux";
+    homepage = "https://github.com/desbma/hddfancontrol";
+    license = licenses.gpl3;
```

Comment:
`gpl3Only` and `gpl3Plus` should be used. https://github.com/NixOS/nixpkgs/blob/ff67dad5711efe6daf0b1a0038b17093617ae502/lib/licenses.nix#L1156

https://www.gnu.org/licenses/identify-licenses-clearly.html

Reply:
The upstream repository does not seem to specify one or the other.  Should I assume `gpl3Only` in this case?

Reply:
I would not make assumption.  Maybe ask the author or leave it unchanged.

Reply:
Seems like you can assume it though: https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206